stdpopsim/slim_engine.py

```diff
@@ -567,57 +608,44 @@ def msprime_rm_to_slim_rm(recombination_map):
     return rates, ends[1:]
 
 
-def get_slim_fractions(contig):
-    return np.array([sum(g.proportions) for g in contig.dfe_list])
```

Comment:
I've deleted these, as the code got much simpler, and I don't think we wanted these methods public anyhow.